src/services/healthcareProfessionalService.ts

```diff
@@ -315,6 +316,15 @@ function validateNames(
         })
     }
 
+   if (!names?.length) {
```

Comment:
I have added this condition because without I could send an empty array and it would still create a healthcare professional. Not sure if this is overkill, since maybe this would be validated in the front?